src/webui/components/AutoComplete/index.js

```diff
@@ -57,6 +57,27 @@ const renderSuggestion = (suggestion, { query, isHighlighted }): Node => {
   );
 };
 
+const renderMessage = (message): Node => {
+  return (
+    <MenuItem selected={false} component="div">
+      <div>{message}</div>
+    </MenuItem>
+  );
+};
+
+const SUGGESTIONS_RESPONSE = {
+  LOADING: 'Loading...',
+  FAILURE: 'Something went wrong.',
+  NO_RESULT: 'No results found.',
+};
+
+const shouldRenderSuggestions = value => {
+  if (value.length > 0) {
+    return true;
+  }
+  return false;
```

Comment:
`value.length > 0` I think you can return directly, not if need it.

Reply:
Removed this method 🛑